codec-http2/src/main/java/io/netty/handler/codec/http2/DefaultHttp2FrameWriter.java

```diff
@@ -537,7 +544,7 @@ private static void verifyStreamOrConnectionId(int streamId, String argumentName
     }
 
     private static void verifyPadding(int padding) {
-        if (padding < 0 || padding > MAX_UNSIGNED_BYTE) {
+        if (padding < 0 || padding > MAX_PADDING) {
```

Comment:
can we share this method? It seems to be used in a few spots. Maybe put it in `Http2CodecUtil`?